FogCreek/Glitch-Community
# Sedate cheese: toggle 'create collection' popover when submitting

## Links
* Remix link: https://sedate-cheese.glitch.me/
* Issue-tracker link: https://app.zenhub.com/workspaces/community-5d07d1359dee6059a9688176/issues/fogcreek/glitch-community-work/389

## Changes:
* toggle popover when adding project to new collection via project popover

## How To Test:
* log into https://sedate-cheese.glitch.me/
* visit any page with project items
* click on a project's options menu
* click "add to collection"
* click "add to a new collection"
* fill in the new collection's name
* click "create"
* the popover should close
* a success notification should indicate that the collection has been created
* visit the new collection's page
* the project you added should be present
